webpack.config.js

```diff
@@ -108,6 +109,8 @@ module.exports = [
                 to: 'media'
             }, {
                 from: 'node_modules/highlightjs/styles/zenburn.css'
+            }, {
+                from: 'assets'
```

Comment:
This will copy the contents of assets to the root of the playground. I.e., assets/stage.png will be available at `localhost:8080/stage.png` not `localhost:8080/assets/stage.png`.  As far as I can see, they need to be available under `/assets/`, so you need to add `to: 'assets'` here.